taskgraph/Task.py

```diff
@@ -405,10 +402,14 @@ def __del__(self):
                         x = self._logging_queue.get_nowait()
                         LOGGER.debug(
                             "the logging queue had this in it: %s", x)
-                    except queue.Empty:
+                    except Exception:
+                        # Normally this could be an empty Queue, but if the
+                        # TaskGraph were being terminated it's possible this
+                        # object is corrupt and we'd get a different kind
+                        # of exception like EOF. In any case we should always
+                        # terminate in the case of an Exception.

```

Comment:
Would it be worth logging the exception here?

Reply:
Sure, i turned the comment into an exception log in 17f521c4e6551daab312444be84343cf401ee5b9